apps/stress-test/benchmarks/browsers.js

```diff
@@ -1,4 +1,4 @@
-const os = require('os');
+// const os = require('os');
```

Comment:
@spmonahan this was throwing a lint error  when trying to merge with master since import is unused so commented it out for now

Reply:
What was the lint error? That package is part of Node so I wouldn't expect any issues.

Reply:
```
'os' is declared but its value is never read.ts(6133)
'os' is assigned a value but never used.eslint[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)